### PR TITLE
Update org references

### PR DESCRIPTION
**Pull request by jasnell**
/cc @mcollina @conoro

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -54,12 +54,12 @@ function writeHtml (data, outputFilename, debug, cb) {
...
   const scriptPath = path.join(__dirname, 'visualizer', 'main.js')
   const logoPath = path.join(__dirname, 'visualizer/assets', 'heap-profiler-logo.svg')
-  const nearFormLogoPath = path.join(__dirname, 'visualizer', 'nearform-logo.svg')
+  // const nearFormLogoPath = path.join(__dirname, 'visualizer', 'nearform-logo.svg')
   const clinicFaviconPath = path.join(__dirname, 'visualizer', 'clinic-favicon.png.b64')
 
   // add logos
   const logoFile = fs.createReadStream(logoPath)
-  const nearFormLogoFile = fs.createReadStream(nearFormLogoPath)
+  // const nearFormLogoFile = fs.createReadStream(nearFormLogoPath)
```
Comment (mcollina) [inline src/index.js:62]: remove

File: src/index.js [hunk 2]
```diff
@@ -92,7 +92,7 @@ function writeHtml (data, outputFilename, debug, cb) {
     headerLogo: logoFile,
     headerText: 'Heap Profiler',
     toolVersion: version,
-    nearFormLogo: nearFormLogoFile,
+    // nearFormLogo: nearFormLogoFile,
```
Comment (mcollina) [inline src/index.js:95]: remove

File: src/index.js [hunk 3]
```diff
@@ -54,12 +54,12 @@ function writeHtml (data, outputFilename, debug, cb) {
   const stylePath = path.join(__dirname, 'visualizer', 'style.css')
   const scriptPath = path.join(__dirname, 'visualizer', 'main.js')
   const logoPath = path.join(__dirname, 'visualizer/assets', 'heap-profiler-logo.svg')
-  const nearFormLogoPath = path.join(__dirname, 'visualizer', 'nearform-logo.svg')
+  // const nearFormLogoPath = path.join(__dirname, 'visualizer', 'nearform-logo.svg')
```
Comment (jasnell) [inline src/index.js:57]:
```suggestion
```

File: src/index.js (hunk 1 above)
Comment (jasnell) [inline src/index.js:62]:
```suggestion
```

File: src/index.js (hunk 2 above)
Comment (jasnell) [inline src/index.js:95]:
```suggestion
```

File: src/visualizer/ui.js [hunk 4]
```diff
@@ -632,17 +632,19 @@ class Ui extends events.EventEmitter {
...
-    return el
+    // el.append('a')
+    //   .classed('nc-header__sponsor', true)
+    //   .attr('href', 'https://nearform.com')
+    //   .attr('title', 'NearForm')
+    //   .attr('target', '_blank')
+    //   .attr('rel', 'noopener noreferrer')
+    //   .html(logo)
+
+    // return el
```
Comment (jasnell) [inline src/visualizer/ui.js:647]:
```suggestion
```

File: src/visualizer/ui.js [hunk 5]
```diff
@@ -615,7 +615,7 @@ class Ui extends events.EventEmitter {
     d3.select('.nc-header__inner').append(() => this.helpButton.button)
 
     // Place sponsor logo bottom left
-    this.footer.d3Element.select('#filters-bar .left-col').append(() => this.createLogoLink().node())
+    // this.footer.d3Element.select('#filters-bar .left-col').append(() => this.createLogoLink().node())
```
Comment (jasnell) [inline src/visualizer/ui.js:618]:
```suggestion
```

File: src/visualizer/ui.js [hunk 6]
```diff
@@ -615,7 +615,7 @@ class Ui extends events.EventEmitter {
     d3.select('.nc-header__inner').append(() => this.helpButton.button)
 
     // Place sponsor logo bottom left
```
Comment (jasnell) [inline src/visualizer/ui.js:617]:
```suggestion
```

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**